README.md

```diff
@@ -327,46 +327,36 @@ Use of this source code is governed by a MIT license that can be found in the [L
 
 ## External sources
 
-In `rv32emu` repository, there are some prebuilt ELF files for testing purpose.
-* `aes.elf` : See [tests/aes.c](tests/aes.c)
-* `captcha.elf` : See [tests/captcha.c](tests/captcha.c)
+There are some prebuilt executables in [rv32emu-prebuilt](https://github.com/sysprog21/rv32emu-prebuilt) for testing and benchmarking purpose.
+The prebuilt executables are built from GNU/GCC v9 and [xPack GCC v14.2.0-1](https://github.com/xpack-dev-tools/gcc-xpack) under Ubuntu-20.04.
```

Comment:
Why did you mention GCC-9?

Reply:
The version of the pre-installed GCC of Ubuntu-20.04 is v9. Maybe upgrade it to v14 through the APT?

Reply:
Most prebuilt ELF files are dedicated to RV32, and the only exception is `tests/arch-test-target/sail_cSim/riscv_sim_RV32`, which serves as the reference simulator for [RISC-V Architecture Test](https://github.com/riscv-non-isa/riscv-arch-test). You shall explain the rationale.

Reply:
The tarball which is going to be released contains the binaries for both amd64 and rv32 in order to automate the benchmark.

Reply:
> The tarball which is going to be released contains the binaries for both amd64 and rv32 in order to automate the benchmark.

Yes, as discussed previously, the prebuilt binary files are provided as artifacts for evaluation and to automate architectural testing.